nncf/openvino/graph/node_utils.py

```diff
@@ -107,48 +107,54 @@ def cnt_if_op(model: ov.Model, cnt: int) -> int:
     return cnt_if_op(model, 0)
 
 
-def get_const_value(const_node: ov.Node, dtype: Optional[np.dtype] = None) -> np.ndarray:
+def get_const_value(const_node: ov.Node, dtype: ov.Type = ov.Type.f32) -> np.ndarray:
     """
     Returns the constant tensor for the node.
 
     :param const_node: OpenVINO node.
-    :param dtype: Destination type.
+    :param dtype: Value return type.
     :return: The constant value.
     """
-    if dtype is None:
-        return const_node.data
-    return const_node.get_data(dtype=dtype)
+    if const_node.get_element_type() == ov.Type.bf16:
+        return const_node.get_data(dtype=dtype.to_dtype())
+    return const_node.data
```

Comment:
I don't get why you removed the conversation to provided `dtype`. I would expect to get a type value based on `dtype` option.

Reply:
Looks reasonable. Returned the previous behaviour.